extension.json

```diff
@@ -112,7 +112,7 @@
 	},
 
 	"ExtensionMessagesFiles": {
-		"WikibaseExportAlias": "WikibaseExport.i18n.alias.php"
```

Comment:
Doesn't matter much, but I think the old name is clearer (esp for people less familiar with MW), and probably purposefully deviated from the MW standard